### PR TITLE
Generate fake data for feed (bug 1131903)

### PR DESCRIPTION
**Pull request by washort**
This should generate the same sort of data as the feed support in https://github.com/mozilla/marketplace-api-mock/

### CODE REVIEW

File: mkt/feed/management/commands/generate_feed.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    Usage:
+
+        python manage.py generate_feed
+
+    """
+
+    help = 'Generate example feed data for development/testing'
+
+    def handle(self, *args, **kwargs):
+        generate_feed_data()
```
Comment (robhudson) [inline mkt/feed/management/commands/generate_feed.py]: Should we add an indexing call here? Or does the model saving trigger indexing via signals?

**Comment by robhudson** [comment]
I glanced briefly, looks fine. I think someone like @ngokevin should look since he's more familiar with feed models.

File: mkt/feed/management/commands/generate_feed.py (hunk 1 above)
Comment (washort) [inline mkt/feed/management/commands/generate_feed.py]: Good question, I was running reindex each time due to the ES problems we discussed earlier. I believe it should index as it goes but I'll check.

**Comment by ngokevin** [comment]
It would be cool if we could share some modules from Marketplace API Mock, but I guess the intention is to get rid of the API Mock. Although the code looks a bit cleaner in the API Mock.

Anyhow, looks fine. Looks like you followed https://github.com/mozilla/marketplace-api-mock/blob/master/factory/feed.py#L217 pretty well.

**Pull request merged**